### PR TITLE
use Homebrew to install libsndfile

### PR DESCRIPTION
**Pull request by rfomin**
MP3 doesn't work in our current AppImage (also there is a crash in Woof 14.3.0, fixed in https://github.com/fabiangreffrath/woof/commit/b12768f41798ad6f75c5a00a448b7c17863f1522 and https://github.com/fabiangreffrath/woof/commit/f43eb49f22d6bb0723afe8ef5ce1279be99d1c3d)

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -90,7 +95,6 @@ jobs:
           CC: ${{ matrix.config.compiler }}
         run: |
           cmake -B build -G Ninja -DCMAKE_BUILD_TYPE=Release \
-                -DCMAKE_INSTALL_PREFIX=/usr \
```
Comment (rfomin) [inline .github/workflows/main.yml:93]: @fabiangreffrath It seems with this line CPack couldn't find Homebrew libs.

Comment (fabiangreffrath) [reply .github/workflows/main.yml:93]: I think this was needed to install/find the soundfont file. 🤔

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -39,6 +39,10 @@ jobs:
             msys-env: mingw-w64-ucrt-x86_64
 
     steps:
+      - name: Set up Homebrew
```
Comment (fabiangreffrath) [inline .github/workflows/main.yml:42]: I'd rather backport the libsndfile package from ubuntu 23.04 than mix in another package manager.

Comment (rfomin) [reply .github/workflows/main.yml:42]: I am probably doing this wrong, but it seems that `libsndfile` is not in the jammy-backports archive? So should we do it ourselves?

**Pull request closed without merging**